schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -60,4 +66,204 @@ public static void main(String... args) {
     }
     System.exit(status);
   }
+
+  /**
+   * Creates tables defined in the schema file.
+   *
+   * @param configProperties Scalar DB config properties.
+   * @param schemaFilePath path to schema file.
+   * @param options specific options for creating tables.
+   * @param createCoordinatorTable create coordinator table or not.
+   */
+  public static void load(
+      Properties configProperties,
+      Path schemaFilePath,
+      Map<String, String> options,
+      boolean createCoordinatorTable)
+      throws SchemaOperatorException {
```

Comment:
Thank you for changing it. But I feel `SchemaOperatorException` is more like an internal exception as `SchemaOperator` is an internally used class and `SchemaOperatorException` is used by it. So I think we should create a new exception like `SchemaLoaderException` and use it here. What do you think?

Reply:
Thanks for discussing and suggesting. I added the `SchemaLoaderException`, applied in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/67a8d4f262d563b8a938889e0ffbaf520c78be4f)